cclib/parser/daltonparser.py

```diff
@@ -76,12 +78,16 @@ def parse_geometry(self, lines):
 
     def extract(self, inputfile, line):
         """Extract information from the file object inputfile."""
-        # extract the version number first
+
+        # Extract the version number and optionally the Git revision
+        # number.
         if line[4:30] == "This is output from DALTON":
-            if line.split()[5] == "release" or line.split()[5] == "(Release":
-                self.metadata["package_version"] = line.split()[6][6:]
-            else:
-                self.metadata["package_version"] = line.split()[5]
+            rs = r"([0-9]{4})\D\s?\w*\s?([0-9]{1})"
+            match = re.search(rs, line)
+            package_version = ".".join(match.groups())
+            self.metadata["package_version"] = package_version
+        if "Last Git revision" in line:
+            revision = line.split()[4]
```

Comment:
Is this ever used?

Reply:
Not yet, see https://github.com/cclib/cclib/issues/533#issuecomment-412240769.